fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckContextualOnboarding.kt

```diff
@@ -26,13 +27,13 @@ import org.mozilla.fenix.compose.LinkText
 import org.mozilla.fenix.compose.LinkTextState
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
 import org.mozilla.fenix.compose.button.PrimaryButton
+import org.mozilla.fenix.shopping.middleware.GetReviewQualityCheckSumoUrl
+import org.mozilla.fenix.shopping.middleware.ReviewQualityCheckNavigationMiddleware
```

Comment:
ui should not depend on middleware.
💭 discussion: maybe we can make the links part of the state or move the constants to store package since it's part of the business logic. This also depends on our approach based on this [comment](https://github.com/mozilla-mobile/firefox-android/pull/4200/files#r1369778333).